kale/loaddata/office_access.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import os
+
+from kale.loaddata.dataset_access import DatasetAccess
+from kale.loaddata.multi_domain import MultiDomainImageFolder
+from kale.prepdata.image_transform import get_transform
+from kale.utils.download import download_file_by_url
+
+url = "https://github.com/sz144/data/raw/main/image_data/office/"
```

Comment:
Do you have a strong reason to host the data? Did you check whether the data owner allows you to host their data? I understand some others (like Jindong Wang) are doing this but typically you should download from the source. 

If you want to do this, you should at least add a readme to acknowledge the source/owner to give them credit and state the reason why you are hosting them rather than downloading from the source.

Would you keep this in your personal GitHub or update to pykale/data?

Reply:
I am using the data from http://www.stat.ucla.edu/~jxie/iFRAME/code/imageClassification.rar. The class names (i.e. folder names) for the caltech domain are not consistent with the other three domains. So I renamed the folders and and uploaded to our pykale/data repo. I also tried the links in https://dalib.readthedocs.io/en/latest/dalib.vision.datasets.html. However there were no response.

Reply:
Good to know. This should be explained but I think the original sources (from Kate?) should be acknowledged and why the original sources are not used should also be explained.

When these are justified, keeping it at pykale/data is better than sz144/data.

Reply:
dalib: your link is outdated. they have renamed to https://github.com/thuml/Transfer-Learning-Library and the docs are at http://170.106.108.162/common/vision/datasets.html